### DISCUSSION TITLE
Lost Important Audacity File AFTER SAVING!

### DISCUSSION THREAD

**Discussion by HildyEric** [audacity/audacity#4926] [Q&A]
### Bug description

Okay, this is going to sound weird, but it happened.
I recorded a great podcast interview on Audacity and, as always, saved the file so I wouldn't lose it. But this time, I left the file open on Audacity thinking that I would cut & paste it into my new project very soon.
I had a forced reboot after that, and Audacity closed down. No big deal, right? When I re-opened Audacity, I upgraded to version 3.2.4. Also no big deal, right? But then I was prompted to "recover" the file I'd saved and left open. Well, I wanted to open that file anyway, so I clicked yes. I then opened a new Audacity file to begin work on my new project. But apparently, I started working in the old file - the one with the important interview.
Here's where it gets weird. The file was blank! Damn it all, but there was no data in the old file, so I mistook it for my new file and began recording. When I was finished and went to save, I realized to my surprise that the file name was the OLD filename already!
I can't find any trace of my lovely interview. Damn it all, but I can't help thinking it's still in my computer somewhere, if I could only find it! I feel like someone has stolen my pet dog. I'm sad, depressed, I'm mourning my loss, and I'm so damned angry I could throw my computer out a window!
Please, if there's any chance I could find this file, somewhere, somehow, I need to know.

### Steps to reproduce

1. Save the project file.
2. Leave the file open.
3. Reboot.
4. "Recover" the "lost" file.

### Expected behavior

The "recovered" file will look like a new, blank project, with no audio content.

### Actual behavior

Exactly as described.

### Audacity Version

latest stable version (from audacityteam.org/download)

### Operating system

Windows

**Comment by petersampsonaudacity** [comment]
Testing on W10 with 3.3.3 and the latest alpha test build I have for the upcoming 3.4.0

I cannot reproduce this - Audacity recovers the saved but left-open project properly.

**Comment by HildyEric** [comment]
I do recall that I also updated my version of audacity, then attempted to
open a new audacity project at the same time as opening the "recovered"
file. Might that explain it?

**Comment by petersampsonaudacity** [comment]
> I do recall that I also updated my version of audacity, then attempted to open a new audacity project at the same time as opening the "recovered" file. Might that explain it?

That should not affect this, no.

**Comment by dozzzzer** [comment]
@HildyEric If you save your file first and then reboot right away, there will be nothing to recover upon start-up, because there were no unsaved changes to your file, and, hence, no force-quitting. So when you launch Audacity after reboot it shows you a blank project screen, from where you should be able to open your project via File > Open. Or am I missing anything here?

Also, FYI, I've tried force-quitting Audacity with some unsaved changes in my project file with a 1h recording, which I was able to recover after reboot.

**Reply by vsverchinsky** [reply]
Correct, but consider
> But this time, I left the file open on Audacity thinking that I would cut & paste it into my new project very soon.

If you do Cut before force-closing Audacity you'll get an empty project upon recovery. Could it be that there's a missing step in the STR?

**Comment by HildyEric** [comment]
You would think. But that's not what happened.
